docs/runbooks/KubeVirtVMIExcessiveMigrations.md

```diff
@@ -0,0 +1,11 @@
+# KubeVirtVMIExcessiveMigrations
+
+## Meaning
+The `KubeVirtVMIExcessiveMigrations` alert means that a VirtualMachineInstance has been live-migrated more than 12 times over a period of 24 hours.  
+This migration rate for a single VMI is significantly higher than a normal operation of KubeVirt, even if taking an upgrade into consideration.
+
+## Impact
+This Alert can suggest an issue in the cluster's underlying infrastructure, e.g. network disruptions, node resource shortage, etc. that causes one VMI or more to be migrated every two hours in average in one day.
+
```

Comment:
@orenc1 you are missing a diagnosis section, where you should explain to the user what and where to look in order to understand what caused the issue. Even what exact commands to run.

Reply:
added a diagnosis section with some commands to check nodes statuses.